semcore/popper/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [5.26.4] - 2024-04-15
+
+### Fixed
+
+- `role` in Trigger, depending on the tag.
```

Comment:
I would write something like

```
### Changed

- The default trigger `role` is `combobox` if an input is used as the trigger, and button otherwise. 
```

Reply:
fixed